src/main/java/org/reactome/release/cosmicupdate/COSMICIdentifierUpdater.java

```diff
@@ -0,0 +1,194 @@
+package org.reactome.release.cosmicupdate;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.model.GKInstance;
+import org.gk.model.InstanceDisplayNameGenerator;
+import org.gk.model.ReactomeJavaConstants;
+import org.gk.persistence.MySQLAdaptor;
+import org.gk.schema.InvalidAttributeException;
+import org.gk.schema.InvalidAttributeValueException;
+import org.reactome.release.common.database.InstanceEditUtils;
+
+/**
+ * Updates a COSMIC Identifier.
+ * Implements <code>Comparable</code> to make sorting easier for reporting purposes. Does *not* override the <code>equals</code> method.
+ * @author sshorser
+ *
+ */
+public class COSMICIdentifierUpdater implements Comparable<COSMICIdentifierUpdater>
+{
+	private static final Logger logger = LogManager.getLogger();
+	private static GKInstance updateWithNewCOSV;
+	private static GKInstance updatePrependCOSM;
```

Comment:
Could this be renamed `instanceEditForPrependCOSMUpdate` so it's explicit it is an `InstanceEdit`?